http-server/src/main/java/org/triplea/server/user/account/PasswordBCrypter.java

```diff
@@ -18,7 +19,7 @@
   public String apply(final String password) {
     // TODO: Md5-Deprecation remove hashing here, move to client-side
     final String hashed = hashPasswordWithSalt(password);
-    return BCrypt.hashpw(hashed, BCrypt.gensalt());
```

Comment:
Are we still using a bcrypt salt anywhere?
If not, for this PR or a fast-follow, we can clean up the configuration that was intended to inject a salt:
- https://github.com/triplea-game/triplea/blob/master/http-server/src/main/java/org/triplea/server/http/AppConfig.java#L57
- https://github.com/triplea-game/triplea/blob/master/http-server/configuration.yml#L12
- https://github.com/triplea-game/triplea/blob/master/infrastructure/ansible/roles/http_server/templates/configuration.yml.j2#L8
- https://github.com/triplea-game/triplea/blob/master/infrastructure/ansible/roles/http_server/defaults/main.yml#L20

Reply:
I'm not sure what you mean by that.
Obviously a salt is still generated for the individual passwords, you just don't have to manually supply them any longer.

Reply:
Good news - That means the `AppConfig` scaffolding added for it, and never wired fully to be used, can be removed as it'll never be used.